File: dayFour/afterBreak/python/taskFive.py
even_sum_total = 0
even_counter = 0
even_counter = int(even_counter)

print("Enter your scores below: ")
for scores in range(1, 11):
    score = int(input())
    if score % 2 == 0:
        even_sum_total = even_sum_total + score
        even_counter += 1
print("The sum of even scores is:", even_sum_total)

File: dayFour/afterBreak/java/TaskFive.java
import java.util.*;


public class TaskFive {
	static Scanner input = new Scanner(System.in);
	
	public static void main() {
	
	int evenScoreSum = 0;
	
	System.out.println("Enter the scores below: ");
	int counter= 0;
	
	for(; counter < 10; counter++)
	  {
	    int score = input.nextInt();
	    if(score % 2 == 0)
	      {
	        evenScoreSum += score;
	      } else { }
	  }
	  
	System.out.println("The sum of even scores is: " + evenScoreSum);
	}
}
